pkg/config/setup/config.go

```diff
@@ -818,6 +818,7 @@ func InitConfig(config pkgconfigmodel.Setup) {
 	config.BindEnv("admission_controller.auto_instrumentation.init_resources.cpu")
 	config.BindEnv("admission_controller.auto_instrumentation.init_resources.memory")
 	config.BindEnv("admission_controller.auto_instrumentation.init_security_context")
+	config.BindEnv("admission_controller.auto_instrumentation.library_storage_medium")
```

Comment:
All of the other flags for this are `init_...` so this is odd being _different_, and also this doesn't apply to the init container but the volume. 

Given we might end up different actual storage drivers, volume sources. Should this be more explicit in its name?

```
auto_instrumentation.volume.empty_dir_storage_medium`
```

I know that's a lot longer, but it's much more explicit to what this does